### ISSUE TITLE
[Task]: Remove Verified/Sponsored promoted tiers

### ISSUE THREAD

**Issue by diox** [mozilla/addons#14996]
### Description

We still have `Verified` and `Sponsored` tiers in addons-server and addons-frontend. We aren't going to use those tiers for the time being, we should remove them to remove confusion.

### Acceptance Criteria

```[tasklist]
  ### Milestones/checkpoints
  - [ ] All code handling sponsored/verified should be removed from addons-server
  - [ ] All code handling sponsored/verified should be removed from addons-frontend
```

### Checks

- [X] If the issue is ready to work on, I have removed the "needs:info" label.
- [X] If I have identified that the work is specific to a repository, I have removed "repository:addons-server" or "repository:addons-frontend"

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/AMOENG-1022)

**Comment by willdurand** [comment]
While "sponsored" is a pure AMO concept, "verified" isn't. It is one of the recommended states that Autograph is able to use: https://github.com/mozilla-services/autograph/blob/19c4068ab8bb2041c913b5212152c7f36478f3cb/autograph.yaml#L1459 (and Firefox too).

IF verified needs to go away, it'd be good to remove it everywhere. Is this a Product call or just some clean-ups?

**Comment by eviljeff** [comment]
> While "sponsored" is a pure AMO concept, "verified" isn't. It is one of the recommended states that Autograph is able to use: https://github.com/mozilla-services/autograph/blob/19c4068ab8bb2041c913b5212152c7f36478f3cb/autograph.yaml#L1459 (and Firefox too).
> 
> IF verified needs to go away, it'd be good to remove it everywhere. Is this a Product call or just some clean-ups?

imo it's low priority clean-up.

**Comment by eviljeff** [comment]
> > imo it's low priority clean-up.
> 
> That didn't answer my question. 

Your question was "Is this a Product call or just some clean-ups?" - it's the latter.

> Was Product aware of the recommendation state file generated at signing time? Was Product OK with getting rid of the "Verified" badge in Firefox as well?

A question for Product (@abyrne-moz )

> Can you link to the bugs filed to fully address the removal in the dependencies (autograph, firefox)?

I'm not aware of any bugs, but they would be dependant on the Product decision, I guess.

**Comment by ioanarusiczki** [comment]
- in dev hub verified/sponsored is no longer displayed on add-ons I've tested with in the past

- admin tools, the promoted module: what was verified/sponsored is now "not promoted", verified or sponsored categories are no longer available. There's : not promoted, recommended, by firefox, spotlight, strategic, notable.

- checked if the remaining categories still work as expected, so I added a new addon id for each. Recommended and By Firefox require manual approval because they're flagged for HR, mozillla-recommendations.json is present with "states":["recommended-android","recommended"] and "states":["line"]. A spotlight is flagged for HR , doesn't have a mozillla-recommendations.json after approval. A strategic is auto-approved. I could also install these add-ons, and see the badges for recommended and by firefox on AMO frontend and addon manager.

- what used to be verified/sponsored when a new version is submitted it is now auto approved. I could install the new version, no badge is displayed on frontend or addon manager but that's https://github.com/mozilla/addons/issues/15038 when I'm going to look on the Primary Hero shelf too on -stage where I've set 2 sponsored add-ons I can check on once this will land there.
